### PR TITLE
Fix a typo in the help file.

### PR DESCRIPTION
**Pull request by LeeWannacott**
Fixed a typo :zany_face:

### CODE REVIEW

File: doc/Colorizer.txt [hunk 1]
```diff
@@ -366,7 +366,7 @@ Note: this may impact startup and (buffer-load) performance.
 Slow performance
 ----------------
 Depending on your file, any of the highlighting functions might cause an
-performance decrease. This can be analyed, by setting the variable
+performance decrease. This can be analysed, by setting the variable
```
Comment (strager) [inline doc/Colorizer.txt:369]: *analyzed

🤡

**Comment by chrisbra** [comment]
Thanks. Can you please change it from *analyse* to *analyze*  as has been pointed out?

**Comment by LeeWannacott** [comment]
Sure, I have changed it to `analyzed` keep in mind `analysed` is not wrong though; it depends where you are in the world:
image

File: doc/Colorizer.txt (hunk 1 above)
Comment (LeeWannacott) [reply doc/Colorizer.txt]: :vomiting_face:

**Comment by chrisbra** [comment]
yes I know and I was a bit unsure about it. But I think because of my education I have a slight preference for American English and that would be the style I tend to use when writing English (even so it's not my native language).

So thanks, appreciate it.

**Pull request merged**